### ISSUE TITLE
Validation set

### ISSUE THREAD

**Issue by feed0** [feed0/german-credit#9]

**Comment by feed0** [comment]
[Training, Testing and Validation Sets](https://stackabuse.com/scikit-learns-traintestsplit-training-testing-and-validation-sets/)
